Mathlib/Analysis/Analytic/Binomial.lean

```diff
@@ -163,3 +163,140 @@ theorem one_add_rpow_hasFPowerSeriesOnBall_zero {a : ℝ} :
 theorem one_add_rpow_hasFPowerSeriesAt_zero {a : ℝ} :
     HasFPowerSeriesAt (fun x ↦ (1 + x) ^ a) (binomialSeries ℝ a) 0 :=
   one_add_rpow_hasFPowerSeriesOnBall_zero.hasFPowerSeriesAt
+
+namespace Complex
```

Comment:
I note that several related lemmas above are in the root namespace. Do you have thoughts on this?

I'd rather be consistent but I'm open to suggestions.

Reply:
I've moved them into a namespace as well.